logging/fb/fluent-bit_config.configmap_azmonitor.yaml

```diff
@@ -900,20 +900,20 @@ data:
        Remove_regex   __temp_.+
     [FILTER]
```

Comment:
When we make changes to the Fluent Bit configuration configMap, we bump the version number (fb_configMap_version) on line 7 and append a string (e.g. "test", or "a").  This makes it easier to confirm the updated configuration is actually being used.  At the end of the sprint, we have a task to remove the additional string and confirm the version number if "correct".  I believe line 7 currently reads: 
`       Set  fb_configMap_version 0.2.19`
So, you might change it to read:
`       Set  fb_configMap_version 0.2.20-test`
It could be argued that the trailing text isn't needed and forces us to remove it in a separate task at the end of the spring, but it is helpful if the file needs to be revised multiple times in a single sprint.